MapboxSearch/sample/src/main/java/com/mapbox/search/sample/AddressAutofillUiActivity.kt

```diff
@@ -160,13 +160,13 @@ class AddressAutofillUiActivity : AppCompatActivity() {
 
     private fun findAddress(point: Point) {
         lifecycleScope.launchWhenStarted {
-            val response = addressAutofill.suggestions(point, AddressAutofillOptions())
+            val response = addressAutofill.reverseGeocoding(point, AddressAutofillOptions())
             response.onValue { suggestions ->
                 if (suggestions.isEmpty()) {
                     showToast(R.string.address_autofill_error_pin_correction)
                 } else {
                     selectSuggestion(
-                        suggestions.first(),
+                        suggestions.first().suggestion,
```

Comment:
It seems it was design error to provide suggestion for results, because we shouldn't allow selection of "already selected" suggestions. As a follow-up we can fix it.

In case of reverse geocoding, it returns results, not suggestion, and we don't need to select them.

Reply:
Fixed